.buildkite/test-pipeline.yaml

```diff
@@ -342,6 +342,19 @@ steps:
   commands:
     - pytest -v -s tool_use
 
+- label: Performance Regression Check
+  fast_check: true
+  working_dir: "/vllm-workspace/.buildkite"
+  source_file_dependencies:
+    - vllm/
+    - csrc/
+  if: build.pull_request != "" # Only run on PRs to check regressions
+  # Additional params to "vllm.entrypoints.openai.api_server" can be provided 
+  # here by PR authors, just remember to delete changes afterwards. 
+  commands:
+    - chmod +x check-performance.sh
+    - bash check-performance.sh --max-model-len 4096
```

Comment:
Might be good to run it a second time here with `VLLM_USE_V1=1`

Reply:
good point!